profiles/tests/test_delete_profile.py

```diff
@@ -77,17 +75,19 @@ def setUp(self) -> None:
                 "person_email": "test3@test.com",
                 "password": "Testing01"
             }).data["auth_token"]
+
     def tearDown(self) -> None:
         self.client.logout()
 
-    def test_get_all_profiles_authorized_no_queryparam(self):
-        response = self.client.get(self.base_url)
-        self.assertEqual(200, response.status_code)
-        self.assertEqual(1, len(response.data))
+    def test_get_profile_unauthorized(self):
+        self.client.logout()
+        response = self.client.get("".join([self.base_url, str(self.user1_id)]))
```

Comment:
I recomment to put explicit URLs in client calls. Using `.format()` method with named placeholders.

So, when looking at the test code, it's super clear, what is the structure of the URL and what the URL actually is.

Reply:
... code duplication is a much lesser evil here. You can enjoy optimizations in other parts of code, but for unittest I recommend to focus on readability. Make unitests code understandable even by non-technical people (it's possible to some extent).

Reply:
Isn't it better to use f-string then so the url will be even more explicit rather than `.format`?